plugins/room/lib/api/KalturaRoomEntry.php

```diff
@@ -81,6 +81,20 @@ public function validateTemplateRoomEntry()
 		}
 	}
 
+	private function retrieveTemplateRoomEntry($entryId)
```

Comment:
Please move this to the appropriate peer class
We shouldn't add DB access methods in the Kaltura class